dbcon/mysql/CMakeLists.txt

```diff
@@ -35,16 +36,17 @@ set_source_files_properties(ha_mcs.cpp PROPERTIES COMPILE_FLAGS "-fno-implicit-t
 
 if (COMMAND mysql_add_plugin)
     mysql_add_plugin(columnstore ${libcalmysql_SRCS} STORAGE_ENGINE MODULE_ONLY DEFAULT
-        LINK_LIBRARIES ${ENGINE_LDFLAGS} ${ENGINE_WRITE_LIBS} ${NETSNMP_LIBRARIES} threadpool
+        LINK_LIBRARIES ${ENGINE_LDFLAGS} ${ENGINE_WRITE_LIBS} ${NETSNMP_LIBRARIES} threadpool aria myisam perfschema
```

Comment:
What exactly do we use from perfschema?

Reply:
I originally had linker errors when trying to link aria symbols, which indirectly depended on symbols from the myisam and perfschema libs. This was when we still had references of ha_maria hton in the code here. Now that we don't explicitly need ha_maria, I found that the linker does not complain anymore. So I have removed this linkage altogether.